src/components/AddTrip/AddTrip.jsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import styles from './AddTrip.module.css';
+import { db } from '../../config/firebase';
+import { addDoc, collection } from 'firebase/firestore';
+import { useNavigate } from 'react-router-dom';
+import { parseTags } from '../../utils/validators';
+import useAuth from '../../contexts/AuthContext';
+import { toast } from 'react-hot-toast';
+
+const AddTrip = () => {
+	const tripsCollectionRef = collection(db, 'Trips');
+
+	const { currentUser } = useAuth();
+
+	const navigate = useNavigate();
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		try {
+			const title = e.target.title.value;
+			const info = e.target?.info.value;
+			const startDate = new Date(e.target.startDate.value);
+			const endDate = new Date(e.target.endDate.value);
+			const startPlace = e.target.startPlace.value;
+			const endPlace = e.target.endPlace.value;
+			const maxParticipantsCount = Number(e.target.maxParticipantsCount.value);
```

Comment:
Skoro owner jest osobno, a participants osobno, to może maxParticipantsCount powinen mieć wartość = to co wprowadzi użytkownik - 1 (owner). Albo od razu do participantsów na indeksie 0 wpisywać UID ownera. Ale wtedy przy wyświetlaniu trzeba będzie pominąć participants[0].

Reply:
**Albo od razu do participantsów na indeksie 0 wpisywać UID ownera**-> zrobione w ten sposób.
Przy dodawaniu uczestników, mozna dopisać maly disclaimer w stylu "pamiętaj, że jako Tworzący Podróż jesteś już jej uczestnikiem", bo biznesowo apka chyba nie zakłada opcji tworzenia podróży dla kogoś/ w czyimś imieniu.